src/test/FuzzerImpl.h

```diff
@@ -22,7 +22,7 @@ struct Operation;
 namespace FuzzUtils
 {
 size_t static constexpr NUM_STORED_LEDGER_KEYS = 0x100U;
-size_t static constexpr NUM_UNVALIDATED_LEDGER_KEYS = 0x40U;
+ssize_t static constexpr NUM_UNVALIDATED_LEDGER_KEYS = 0x40;
```

Comment:
`ssize_t` is not what you want (this is a C standard type not C++). I imagine that you're trying to silence the warning in `generateStoredLedgerKeys`?

If so the type should be `StoredLedgerKeys::iterator::difference_type` (on my machine this is `ptrdiff_t` but I don't know if this is true across platforms)

Reply:
Fixed (it's just `StoredLedgerKeys::difference_type`)